web-tech-exercises/4/style.css

```diff
@@ -0,0 +1,315 @@
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+
+body {
+  font-family: 'Roboto', sans-serif;
+}
+
+
+.container {
+  display: grid;
+  height: 100vh;
+  grid-template-columns: 20% auto;
+  background-color: #f5f5f5;
+}
+
+
+aside {
+  border-right: 1px solid rgb(205, 205, 205);
+  background-color: #ffffff;
+}
+
+
+.um-logo {
+  height: 100px;
+  padding: 16px 24px;
+  border-bottom: 1px solid rgb(205, 205, 205);
+  background-color: #ffffff;
+  width: 100%;
+}
+
+
+.um-user {
+  padding: 16px 24px;
+  border-bottom: 1px solid rgb(205, 205, 205);
+  display: flex;
+  align-items: center;
+}
+
+
+.um-user img {
+  width: 40px;
+  border-radius: 50%;
+  margin-right: 12px;
+}
+
+
+.um-user-name {
+  color: #007acc;
+  display: block;
+}
+
+
+.um-user-email {
+  font-size: x-small;
+  color: #3b3b3c;
+}
+
+.nav-menu {
+  padding: 24px;
+  list-style: none;
+}
+
+
+.nav-menu li {
+  margin-bottom: 16px;
+}
+
+.nav-menu li a {
+  text-decoration: none;
+  color: #007acc;
+  font-size: 14px;
+}
+
+.um-navitems__main {
+  list-style-type: none;
+  padding: 0%;
+}
+
+.um-navitems__main li {
+  padding: 10px;
+  color: #007acc;
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.um-navitems__main li i {
+  font-size: 24px;
+  padding: 15px;
+}
+
+
+.um-navitems__secondary {
+  list-style-type: none;
+  padding: 0%;
+}
+
+.um-navitems__secondary li {
+  padding: 10px;
+  color: #007acc;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.um-navitems__secondary li i {
+  font-size: 24px;
+  padding: 15px;
+}
+
+
+.test-data > span:first-of-type {
+  background-color: black;
+  color: white;
+  padding: 3px;
+}
+
+
+.test-data > span,
+.test-data > i {
+  background-color: #084f7e;
+  color: white;
+  padding: 3px;
+}
+
+
+.test-data > i:last-of-type {
+  margin-left: auto;
+}
+
+
+.um-test-name {
+  color: white;
+  background-color: #084f7e;
+  padding: 3px;
+}
+
+
+.test-properties {
+  display: flex;
+  flex-direction: column;
+}
+
+
+.test-properties > span:first-of-type {
+  background-color: #007acc;
+  color: white;
+  padding: 2px;
+}
+
+
+.form-properties {
+  display: flex;
+  flex: 1;
+}
+
+
+.details {
+  flex: 1;
+  padding: 10px;
+  border: 1px solid #ddd;
+}
+
+.description {
+  flex: 1;
+  padding: 10px;
+  border: 1px solid #ddd;
+}
```

Comment:
Styles for `.details` and `.description` can be combined

Reply:
done ✅